lighthouse-cli/bin.ts

```diff
@@ -330,8 +330,14 @@ export async function runLighthouse(url: string,
     const chromeLauncher = await getDebuggableChrome(flags)
     const results = await lighthouse(url, flags, config);
 
+    // Filter out artifacts except for screenshots in traces to minimize report size.
+    var traces = results.artifacts.traces;
+    for (var pass in traces)
+      traces[pass]['traceEvents'] = traces[pass]['traceEvents'].filter(e => e['cat'] === 'disabled-by-default-devtools.screenshot');
```

Comment:
+1 on fixing speedline perf, but let's not block the filmstrip here on it. 

*nod* on a dedicated screenshots audit. seems good to have it top-level than hide it in ext-info.